src/server/api/controllers/EmailController.ts

```diff
@@ -0,0 +1,41 @@
+import { AuthorisedAdmin } from '../decorators/AuthorisedAdmin';
+import { SelectedEvent } from '../decorators/SelectedEvent';
+import {AdminAuthorisation} from '../middleware/AdminAuthorisation';
+import { RoleAuth } from '../middleware/RoleAuth';
+import { ValidateEventAlias } from '../middleware/ValidateEventAlias';
+import { Logger } from '@Config/Logging';
+import EmailService from '@Services/EmailService';
+import { Role } from '@Shared/Roles';
+import { AcceptanceEmailRequest } from '@Shared/api/Requests'
+import { EventDocument } from '@Models/Event';
+import { Post, JsonController, UseBefore, Req, Body } from 'routing-controllers';
+import { Request } from 'express';
+import { Admin } from '@Shared/ModelTypes';
+import { SuccessResponse } from '@Shared/api/Responses';
+
+@JsonController('/emails')
+@UseBefore(AdminAuthorisation)
+export class EmailController {
+  constructor(
+    private EmailService: EmailService
+  ) {}
+
+  @Post('/acceptance/:eventAlias')
+  @UseBefore(RoleAuth(Role.ROLE_ADMIN))
+  @UseBefore(ValidateEventAlias)
+  async sendAcceptanceEmail(
+    @AuthorisedAdmin() admin: Admin,
+    @Req() request: Request,
+    @SelectedEvent() event: EventDocument,
+    @Body() body: AcceptanceEmailRequest
+  ): Promise<SuccessResponse> {
+    const sendGridResponse = await this.EmailService.sendEventAcceptanceEmail(request, admin, event, body.userEmail);
```

Comment:
We can get the event alias from the user in the body of the request - better to leave it out of the URL.
Also want to check that the admin is an organiser of the event (there's a call for this in the EventService).

Reply:
Why? Then I won't be able to use the `isOrganiser` middleware and would have to code-duplicate a bunch of `EventService.isAdminOrganiser` calls

Reply:
I'll fix this in my refactoring branch once this merges. I have new methods